### ISSUE TITLE
hoedown -> pulldown regression: missing whitespace in multi-line bullet points

### ISSUE THREAD

**Issue by colin-kiegel** [rust-lang/rust#41162]
@steveklabnik this seems to be a hoedown -> pulldown regression #40912 

# IN

```rust
//! * lorem ipsum
//!   dolor sit
//!   amet
//! * lorem ipsum dolor sit amet
```

# OUT

## 1.18.0-nightly (2017-04-06)

```markdown
* lorem ipsumdolor sitamet
* lorem ipsum dolor sit amet
```

## 1.15.0 ... 1.17.0-beta.3 (2017-04-05)

```markdown
* lorem ipsum dolor sit amet
* lorem ipsum dolor sit amet
```

**Comment by colin-kiegel** [comment]
Interestingly this does not seem to be a bug in pulldown-cmark, but maybe something post-processing related?

The doc-output of the nightly compiler is
```html
<ul><li>lorem ipsumdolor sitamet</li><li>lorem ipsum dolor sit amet</li></ul>
```

While pulldown-cmark itself outputs:
```html
<ul>
<li>lorem ipsum
dolor sit
amet</li>
<li>lorem ipsum dolor sit amet</li>
</ul>
```

<details>
<summary>Example Code</summary>
<pre lang="rust"><code>
//! * lorem ipsum
//!   dolor sit
//!   amet
//! * lorem ipsum dolor sit amet
extern crate pulldown_cmark;
use pulldown_cmark::{html, Parser};

const MARKDOWN: &'static str = r##"
\* lorem ipsum
  dolor sit
  amet
\* lorem ipsum dolor sit amet
"##;

fn main() {
&emsp; mut buf = String::new();
&emsp; let parser = Parser::new(MARKDOWN);
&emsp; html::push_html(&mut buf, parser);

&emsp; println!("{}", buf);
}
</code></pre>
</details>

**Comment by ollie27** [comment]
This was fixed by #41112. You'll need to update to the latest nightly.

**Comment by colin-kiegel** [comment]
Right. That was a quick fix - thanks. :-)
